src/lib/openapi/spec/admin-features-query-schema.ts

```diff
@@ -0,0 +1,26 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const adminFeaturesQuerySchema = {
+    $id: '#/components/schemas/adminFeaturesQuerySchema',
+    type: 'object',
+    additionalProperties: true,
```

Comment:
What is this schema used for? Is there anything the user should be aware of about the schema in general?

```suggestion
    additionalProperties: true,
    description: 'todo',
```

Reply:
Thank you Thomas. Good point. I'll update the docs.